lib/sms_processors.rb

```diff
@@ -0,0 +1,34 @@
+require 'pry'
+
+class SMSProcess
```

Comment:
I think this is just for the Sinatra experiment, but it feels like the Class has several responsibilities. I think completing this week would help you adjust your approach.